pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -360,6 +361,9 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 
 	log.Print("Ensuring pods security group")
 	podSgId, err := ensureOpenStackSg(client, generateName("kuryr-pods-security-group", clusterID), tag)
+	if err != nil {
+		return nil, errors.Wrap(err, "pods security group is not present")
```

Comment:
"failed to create pods security group" would be more suitable here.

Nice catch by the way.